src/components/ec-input-field/ec-input-field.spec.js

```diff
@@ -155,4 +156,9 @@ describe('EcInputField', () => {
     const wrapper = mountInputField({ icon: 'simple-check' }, { attrs: { disabled: true } });
     expect(wrapper.element).toMatchSnapshot();
   });
+
+  it('renders properly when the given prop is set', () => {
+    const wrapper = mountInputField({ labelTooltip: 'Testing the labelTooltip prop' });
+    expect(wrapper.props().labelTooltip).toBe('Testing the labelTooltip prop');
```

Comment:
This test doesn't check the logic in the template (`v-if="labelTooltip"`) it tests that Vue can set props. That prop might not be used properly in the template or in computed props or other methods. We should check what was rendered in the wrapper by checking:
```
expect(wrapper.element).toMatchSnapshot();
```
We want to see that the icon is not there in all other tests except for this specific one.